src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -7,7 +7,7 @@
         {% set is_authenticated = collector.enabled and collector.authenticated  %}
         {% set color_code = is_authenticated ? '' : 'yellow' %}
```

Comment:
this one would be updated also (maybe other similar?)

Reply:
Done + other similar